client/components/pie-chart/index.js

```diff
@@ -8,9 +8,9 @@ import DataType from './data-type';
 
 import './style.scss';
 
-const SVG_SIZE = 300;
+const SVG_SIZE = 224;
```

Comment:
@dognose24 do you recall if any other place was set to 300px? I can see that CSV viewBox is set to 224 but the chart is still 300 x 300 px. Maybe it's caused by height set somewhere but I can't find it.